webpack.test.bootstrap.ts

```diff
@@ -0,0 +1,2 @@
+let testContext = (<{ context?: Function }>require).context('./', true, /\.test\.ts/);
```

Comment:
@bertrandk this is my code :) I pointed @vanessayuenn to it as an example.
This is equivalent of saying find all the test files, and call `require` on each dynamically so as to make sure they are all a big happy family within the bundle generated by tests